docker/Dockerfile

```diff
@@ -2,10 +2,13 @@
 # climate-explorer-backend/docker/Dockerfile.dev
 # As always with docker, is there a better way to do this? Almost certainly.
 
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
 MAINTAINER Rod Glover <rglover@uvic.ca>
```

Comment:
This label has been deprecated, it should be updated to `LABEL Maintainer="Rod Glover <rglover@uvic.ca>"`